src/tests/specs/iteration.spec.ts

```diff
@@ -33,13 +33,16 @@ describe('Iteration test', () => {
     await planner.sidePanel.ready();
     await planner.workItemList.workItem(workItemTitle1).openQuickPreview();
     await planner.quickPreview.addIteration(dropdownIteration1);
+    await planner.quickPreview.close();
```

Comment:
Do we still need this?

Reply:
Yes we need this, we are adding/updating the iteration Text and then validating added/updated iteration text in the table view which is hidden by quick Preview.

Reply:
Okay. Thanks